ttnn/ttnn/operations/core.py

```diff
@@ -580,7 +579,7 @@ def from_torch_and_dump(
             )
             pathlib.Path(cache_file_name).parent.mkdir(parents=True, exist_ok=True)
             distributed_config = mesh_mapper.config() if mesh_mapper else dict()
-            ttnn.dump_tensor(cache_file_name, tensor, distributed_config)
```

Comment:
Why this change happened? Is it intentional?

Reply:
previously ttnn.dump_tensor api  just called tt_lib.tensor.dump_tensor api
now that the binding for dump_tensor is moved to a ttnn submodule, makes sense to use it.

Reply:
But can we expose `dump_tensor` from `ttnn._ttnn.tensor` into `ttnn`?